SplunkAppForWazuh/appserver/static/js/controllers/management/configuration/edit-configuration/editRulesetCtrl.js

```diff
@@ -286,13 +291,19 @@ define([
       try {
         delete this.scope.currentList.list[key]
         this.scope.removingEntry = false
-        await this.saveList()
+        this.refreshCdbList()
       } catch (error) {
         this.toast("Error deleting entry.")
       }
 
     }
 
+    async refreshCdbList() {
+      this.scope.items = this.cdbToArr()
+      this.initPagination()
```

Comment:
Is `this.cdbToArr()` async? Is `this.initPagination()` async? if not the case, `refreshCdbList` must not be async. On the other hand, if one of them are async, you must use `try/catch` + `await`.

Thanks!